src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -69,7 +71,7 @@ class Repository(
             visit.visitors.addAll(
               visitBuilder.visitors.map {
                 it.build(it.person, leadVisitor = it.leadVisitor, visit)
-              }
+              } + listOf(visitBuilder.visitOutcome.build(visit, visitBuilder.visitOutcome.outcomeCode?.let { lookupVisitOutcomeReason(it) }))
             )
```

Comment:
in NOMIS the outcome is modelled as an extra dummy visitor, so we add it to the list of visitors